org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/MultiRootTest.java

```diff
@@ -41,6 +42,8 @@ public void testInitializeWithMultiFolders() throws Exception {
 			assertEquals(3, WorkspaceHelper.getAllProjects().size()); // includes the default project
 			assertNotNull(WorkspaceHelper.getProject("hello"));
 			assertNotNull(WorkspaceHelper.getProject("salut"));
+			WorkspaceHelper.deleteAllProjects();
+			FileUtils.forceDelete(getWorkingProjectDirectory());
```

Comment:
mmm you're not supposed to delete the projects here. This test checks whether the change of multi root is gracefully supported

Reply:
The test shouldn't delete the salut project.